Runtime/LODGenerator.cs

```diff
@@ -166,11 +166,16 @@ public static LODGroup GenerateLODs(GameObject gameObject, LODLevel[] levels, bo
                 {
                     var meshRenderers = (from renderer in originalLevelRenderers
                                          where renderer.enabled && renderer as MeshRenderer != null
+                                        && renderer.transform.GetComponent<MeshFilter>() != null
+                                        && renderer.transform.GetComponent<MeshFilter>().sharedMesh != null
                                          select renderer as MeshRenderer).ToArray();
                     var skinnedMeshRenderers = (from renderer in originalLevelRenderers
                                                 where renderer.enabled && renderer as SkinnedMeshRenderer != null
+                                                && renderer.transform.GetComponent<SkinnedMeshRenderer>().sharedMesh != null
```

Comment:
This could be simplified to (renderer as SkinnedMeshRenderer).sharedMesh instead of getting the component again.

Reply:
Hi, sorry again so careless of me. I did the requested changed.